src/router.js

```diff
@@ -2,27 +2,51 @@ import Vue from 'vue'
 import Router from 'vue-router'
 import Register from './views/Register.vue'
 import Login from './views/Login.vue'
-import NavBar from './components/NavBar.vue'
+import BookList from './views/BookList.vue'
 import { routes } from './routes'
+import { isAuth } from './config/api'
 
 Vue.use(Router)
 
+const ifNotAuthenticated = (to, from, next) => {
```

Comment:
When you don use arguments define it with '_'
```suggestion
const ifNotAuthenticated = (_, _, next) => {
```